dbt/manifest/v12.json

```diff
@@ -727,12 +727,6 @@
                 "type": "string",
                 "default": ""
               },
-              "vars": {
```

Comment:
Are these expected?

Reply:
Yes. `vars` was reverted in core. Core team discussed it and decided it was safe to remove from the jsonschema